templates/tom_targets/partials/aladin.html

```diff
@@ -0,0 +1,179 @@
+{% load tom_common_extras targets_extras target_list_extras %}
+<!-- include Aladin Lite CSS file in the head section of your page -->
+<link rel="stylesheet" href="//aladin.u-strasbg.fr/AladinLite/api/v2/latest/aladin.min.css" />
+<!-- insert this snippet where you want Aladin Lite viewer to appear and after the loading of jQuery -->
+<h4 style='color:#FF69B4'>Survey View</h4>
```

Comment:
If I understand this correctly, the only change here is to make the text pink. If that's the case, you should do it using a style sheet (CSS) instead of directly in the HTML. We want to avoid duplicating code as much as possible.